apps/api/src/app/blueprint/usecases/get-grouped-blueprints/get-grouped-blueprints.usecase.ts

```diff
@@ -51,13 +51,22 @@ export class GetGroupedBlueprints {
     groups: { name: string; blueprints: NotificationTemplateEntity[] }[]
   ): INotificationTemplate[] {
     const storedBlueprints = this.extractStoredBlueprints(groups);
+    const firstBlueprint = storedBlueprints[0];
 
     const localPopularBlueprints = { ...POPULAR_TEMPLATES_GROUPED };
 
     return localPopularBlueprints.blueprints.map((localBlueprint) => {
       const storedBlueprint = storedBlueprints.find((blueprint) => blueprint._id === localBlueprint._id);
 
-      return (storedBlueprint ? storedBlueprint : localBlueprint) as INotificationTemplate;
+      return (
+        storedBlueprint
+          ? storedBlueprint
+          : {
+              ...localBlueprint,
+              _notificationGroupId: firstBlueprint._notificationGroupId,
+              notificationGroup: firstBlueprint.notificationGroup,
+            }
+      ) as INotificationTemplate;
```

Comment:
Basically, the popular blueprints are hardcoded constants that don't have the notification group and this raises an API error when creating the workflow from it on the FE... 

I'm not sure what is the best way to bypass this, but here I'm assigning the first blueprint notification group...

@djabarovgeorge cc

Reply:
Due to our decision last week (don't use hardcoded data directly on the client) the only thing that we will need is the ID, so we won't have this issue.